frontend/app/src/components/display/AmountDisplay.vue

```diff
@@ -13,17 +13,17 @@
         {{
           renderValue
             | calculatePrice(exchangeRate(currency.ticker_symbol))
-            | roundDown(floatingPrecision)
+            | formatPrice(amountDisplayFormat, floatingPrecision)
         }}
       </span>
       <span
         v-else-if="fiatCurrency === currency.ticker_symbol"
         class="amount-display__value"
       >
-        {{ renderValue | roundDown(floatingPrecision) }}
+        {{ renderValue | formatPrice(amountDisplayFormat, floatingPrecision) }}
       </span>
       <span v-else class="amount-display__value">
-        {{ renderValue | formatPrice(floatingPrecision) }}
+        {{ renderValue | formatPrice(amountDisplayFormat, floatingPrecision) }}
```

Comment:
This the `else-if` and `else` clauses here now do the same thing. Previously this (and the original v-if) determined whether we were rounding up/down.
```
      <span
        v-else-if="fiatCurrency === currency.ticker_symbol"
        class="amount-display__value"
      >
        {{ renderValue | formatPrice(amountDisplayFormat, floatingPrecision) }}
      </span>
      <span v-else class="amount-display__value">
        {{ renderValue | formatPrice(amountDisplayFormat, floatingPrecision) }}
      </span>
```

Reply:
I would guess we will still want the old behaviour when it comes to rounding, since we already had lengthy discussions on that and decided on this approach.

Also for the currency display, I agree it would make sense to have the handling as part of the amount display.

Reply:
I wasn't aware of this specific behaviour, so I have to get back that difference using formatPrice 👍